roles/openstack-meta/defaults/main.yml

```diff
@@ -398,6 +398,233 @@ openstack_meta:
           config_files: /etc/neutron/neutron.conf,/etc/neutron/services/loadbalancer/haproxy/lbaas_agent.ini,/etc/neutron/neutron_lbaas.conf
           restart: on-failure
           kill_mode: process
+  swift:
+    services:
+      swift_account:
+        ubuntu:
+          name: swift-account
+          desc: OpenStack Object Storage (swift) - Account Service
+          cmd: /usr/local/bin/swift-account-server
+          config_files: /etc/swift/account-server.conf
+          user: swift
+        rhel:
+          name: openstack-swift-account
+          desc: OpenStack Object Storage (swift) - Account Service
+          cmd: /usr/bin/swift-account-service
```

Comment:
this comment is applicable to all the swift services under rhel:  this is needed to handle centos case where is is installed from source.

instead of just having cmd: /usr/bin/swift-account-service
cmd: "{{ (ansible_distribution == 'RedHat') | ternary('/usr/bin/', '/usr/local/bin/') }}swift-account-service"